panic: runtime error: slice bounds out of range [24:3]

I have just found this panic file that apparently took place today at night. I wasn't even aware of the fact, so Syncthing must have just restarted itself automatically and continued to work as usual. I also haven't found anything special in the system logs that could be connected with this.

```
Panic at 2021-04-13T02:12:53+09:00
panic: runtime error: slice bounds out of range [24:3]

goroutine 98846 [running]:
github.com/ccding/go-stun/stun.newPacketFromBytes(0xc004f5c400, 0x44, 0x400, 0x400, 0x44, 0x122fb48)
	github.com/ccding/go-stun@v0.1.2/stun/packet.go:60 +0x574
github.com/ccding/go-stun/stun.(*Client).send(0xc0005981c0, 0xc0023c2740, 0x1242a48, 0xc000598180, 0x122fb48, 0xc00cbedf80, 0x3624c5, 0xc00d133f40, 0x0)
	github.com/ccding/go-stun@v0.1.2/stun/net.go:90 +0x1c5
github.com/ccding/go-stun/stun.(*Client).sendBindingReq(0xc0005981c0, 0x1242a48, 0xc000598180, 0x122fb48, 0xc00cbedf80, 0xc00cbd0000, 0x10, 0x10, 0x0)
	github.com/ccding/go-stun@v0.1.2/stun/net.go:54 +0x433
github.com/ccding/go-stun/stun.(*Client).test1(...)
	github.com/ccding/go-stun@v0.1.2/stun/tests.go:24
github.com/ccding/go-stun/stun.(*Client).discover(0xc0005981c0, 0x1242a48, 0xc000598180, 0xc00cbedf80, 0xc00cbedf80, 0x0, 0x0, 0x1)
	github.com/ccding/go-stun@v0.1.2/stun/discover.go:72 +0x116
github.com/ccding/go-stun/stun.(*Client).Discover(0xc0005981c0, 0x0, 0x0, 0x0, 0x0)
	github.com/ccding/go-stun@v0.1.2/stun/client.go:101 +0xfc
github.com/syncthing/syncthing/lib/stun.(*Service).runStunForServer.func1(0x0, 0x384686)
	github.com/syncthing/syncthing/lib/stun/stun.go:189 +0x54
github.com/syncthing/syncthing/lib/util.CallWithContext.func1(0xc00cbedf20, 0xc00d133f30, 0xc00fe61740)
	github.com/syncthing/syncthing/lib/util/utils.go:257 +0x2e
created by github.com/syncthing/syncthing/lib/util.CallWithContext
	github.com/syncthing/syncthing/lib/util/utils.go:256 +0x99
````

[panic-20210413-021253.log](https://github.com/syncthing/syncthing/files/6302279/panic-20210413-021253.log)

This is Syncthing v1.15.0 under Windows 10 (as usual, custom compiled with a few minor tweaks).

**Edit**: I'm sorry for creating the issue with no bug label again, but this is on GitHub not giving any choices unless I log in before pressing the new issue button, which I seem to forget to do.

--- a/go.mod
+++ b/go.mod
@@ -6,7 +6,7 @@ require (
 	github.com/alecthomas/kong v0.2.12
 	github.com/bkaradzic/go-lz4 v0.0.0-20160924222819-7224d8d8f27e
 	github.com/calmh/xdr v1.1.0
-	github.com/ccding/go-stun v0.1.2
+	github.com/ccding/go-stun v0.1.3
 	github.com/certifi/gocertifi v0.0.0-20200922220541-2c3bb06c6054 // indirect
 	github.com/chmduquesne/rollinghash v0.0.0-20180912150627-a60f8e7142b5
 	github.com/d4l3k/messagediff v1.2.1

